_posts/2021-02-11-highs.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "HiGHS.jl 0.1 released"
+date:   2021-02-11
+categories: [solvers]
+---
+
+We are happy to announce the initial `v0.1` release of [HiGHS.jl](https://github.com/jump-dev/HiGHS.jl),
+a wrapper for [HiGHS](https://www.maths.ed.ac.uk/hall/HiGHS/).
+
+HiGHS is a new high-performance open-source linear programming solver being
+developed by Julian Hall and colleagues at the University of Edinburgh,
+Scotland.
+
+You can try it out as follows:
+```julia
+import Pkg; Pkg.add("HiGHS")
+
+using JuMP, HiGHS
+model = Model(HiGHS.Optimizer)
+set_silent(model)
+```
+
+While HiGHS is ready to use today, you should be aware that it is still under
+active development. That means it might be slow or unstable on some problems.
+However, by reporting these problems you can help make HiGHS better!
+
+To report a problem (e.g., incorrect results, or a crash of the solver),
+or make a suggestion for how to improve HiGHS, please [file a GitHub issue](https://github.com/jump-dev/HiGHS.jl).
+To make the problem reproducible, use `JuMP.write_to_file(model, "filename.mps")`
```

Comment:
Perhaps these instructions would be more visible either in the HIGHS.jl readme or in the HIGHS.jl issue template.

Reply:
Added to the README as well: https://github.com/jump-dev/HiGHS.jl/pull/34.